api/graphql/schema.graphql

```diff
@@ -520,29 +520,160 @@ type Query {
     skills: [Skill]! @all
     experienceSkills: [ExperienceSkill]! @all
 }
+input ClassificationBelongsTo {
+    connect: ID!
+}
+input PoolsHasMany {
+    create: [CreatePoolInput]
+}
 """
 When creating a User, name and email are required.
 """
 input CreateUserInput {
     sub: String
+    roles: [Role] = []
+
+    # Personal info
     firstName: String! @rename(attribute: "first_name")
     lastName: String! @rename(attribute: "last_name")
     email: Email! @rules(apply: ["unique:users,email"])
     telephone: PhoneNumber
+    currentProvince: ProvinceOrTerritory @rename(attribute: "current_province")
+    currentCity: String @rename(attribute: "current_city")
+
+    # Language
+    languageAbility: LanguageAbility @rename(attribute: "language_ability")
     preferredLang: Language @rename(attribute: "preferred_lang")
-    roles: [Role] = []
+    lookingForEnglish: Boolean @rename(attribute: "looking_for_english")
+    lookingForFrench: Boolean @rename(attribute: "looking_for_french")
+    lookingForBilingual: Boolean @rename(attribute: "looking_for_bilingual")
+    bilingualEvaluation: BilingualEvaluation @rename(attribute: "bilingual_evaluation")
+    comprehensionLevel: EvaluatedLanguageAbility @rename(attribute: "comprehension_level")
+    writtenLevel: EvaluatedLanguageAbility @rename(attribute: "written_level")
+    verbalLevel: EvaluatedLanguageAbility @rename(attribute: "verbal_level")
+    estimatedLanguageAbility: EstimatedLanguageAbility @rename(attribute: "estimated_language_ability")
+
+    # Gov info
+    isGovEmployee: Boolean @rename(attribute: "is_gov_employee")
+    interestedInLaterOrSecondment: Boolean @rename(attribute: "interested_in_later_or_secondment")
+    currentClassification: ClassificationBelongsTo
+
+    # Employment equity
+    isWoman: Boolean @rename(attribute: "is_woman")
+    hasDisability: Boolean @rename(attribute: "has_disability")
+    isIndigenous: Boolean @rename(attribute: "is_indigenous")
+    isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
+
+    # Applicant info
+    jobLookingStatus: JobLookingStatus @rename(attribute: "job_looking_status")
+    pools: PoolsHasMany #Pools that a user owns
```

Comment:
could we create a pool & pool candidate that user owns when we create the user himself for the first time ?  I'm not sure we need this here.

Reply:
`interestedClassifications: relationship to many Classifications` is missing here